src/Microsoft.DotNet.Build.Tasks/PackageFiles/tool-runtime/project.json

```diff
@@ -20,7 +20,7 @@
       "version": "4.0.0-rc2-23907",
       "exclude": "Compile"
     },
-    "MSBuild": "0.1.0-preview-00017",
+    "MSBuild": "0.1.0-preview-00018",
```

Comment:
@cdmihai, should this PR be closed, since the current master has newer version ->`"MSBuild": "0.1.0-preview-00022"`.